src/osd/OSD.cc

```diff
@@ -2827,31 +2827,25 @@ void OSD::load_pgs()
     derr << "failed to list pgs: " << cpp_strerror(-r) << dendl;
   }
 
-  set<spg_t> pgs;
```

Comment:
Can you move the removal of this into a separate patch?  It's logically unrelated to the other change.